src/pages/Player.vue

```diff
@@ -57,7 +60,6 @@
           :previewMode="false"
           :isModalMinimized="isModalMinimized"
           :isFullscreen="isFullscreen"
-          :isPortrait="isPortrait"
```

Comment:
why change how this value is passed?
and if it has been deleted, wherever it's been defined in this file should also be removed right?